### ISSUE TITLE
[Re] Learning with Noisy Labels Revisited: A Study Using Real-World Human Annotations

### ISSUE THREAD

**Issue by ValterH** [ReScience/submissions#86]
**Original article:** [Learning with Noisy Labels Revisited: A Study Using Real-World Human Annotations](https://openreview.net/pdf?id=TBWA6PLJZQm)

**PDF URL:**  https://github.com/KlemenVovk/noisy-labels/blob/master/rescience/rescience.pdf
**Metadata URL:**  https://github.com/KlemenVovk/noisy-labels/blob/master/rescience/metadata.yaml
**Code URL:**  https://github.com/KlemenVovk/noisy-labels
**Openreview:** https://openreview.net/forum?id=GKZ2leags0

**Scientific domain:**  Machine Learning
**Programming language:**  Python
**Suggested editor:**  @rougier

**Comment by rougier** [comment]
@ValterH Very sorry for such long delay, for some reasons, I did not see your submission.
Is your paper part of the ML Reproducibility Challenge ? If yes, do you have open peer review already?

**Comment by ValterH** [comment]
@rougier No problem. We were under the impression that there was no reproducibility challenge for 2024, so we decided to submit to ReScience C directly.

**Comment by rougier** [comment]
Ok, and sorry for long delay in my answer, end of year was pretty chaotic. I'll handle your submission.

**Comment by rougier** [comment]
@mo-arvan @MollyZhang Coudl you review this submission?

**Comment by rougier** [comment]
@ValterH Don't hesitate to notify me here if things are not moving. Else, you can also mail me directly.

**Comment by ValterH** [comment]
@rougier I understand, no worries. Thank you for moving things along!

**Comment by rougier** [comment]
@gdetor @mo-arvan @MollyZhang @pena-rodrigo Could you review this submission?

**Comment by ValterH** [comment]
We decided to submit to [MLRC 2025](https://reproml.org/call_for_papers/).

**Comment by ValterH** [comment]
@rougier I am reopening our submission. I added links to the updated article and paper reviews from MLRC.

**Comment by rougier** [comment]
Ok, are there any public reviews by any chance?
**[Just realized you added the link to open reviews...]**

**Comment by ValterH** [comment]
> Ok, are there any public reviews by any chance?

Yes, the reviews are available at https://openreview.net/forum?id=GKZ2leags0

**Comment by rougier** [comment]
Thanks. Based on these public reviews, I think we can accept directly your paper for publication.

 My only two comments concerns the title: [Re] means full replication, [~Re] means partial replication and [¬Re] means not reproducible. Based on your discussion / conclusion, do you prefere to keep [Re] or to use [~Re] ? the other very minor comment is in the text with a lot line returns. Is that intended ?

**Comment by ValterH** [comment]
Thank you for the quick response!

> My only two comments concerns the title: [Re] means full replication, [~Re] means partial replication and [¬Re] means not reproducible. Based on your discussion / conclusion, do you prefere to keep [Re] or to use [~Re] ?

I agree that a partial reproduction is more approporiate, hence we will change the name to use  [~Re].

> the other very minor comment is in the text with a lot line returns. Is that intended ?

I added breaks between paragraphs to replace the line returns. 

Both changes are reflected in the updated version of the article: https://github.com/KlemenVovk/noisy-labels/blob/rescience/rescience/rescience.pdf

**Comment by ValterH** [comment]
@rougier Is there any further action needed on our end?

**Comment by rougier** [comment]
Sorry for long delay, end of year was a bit crazy. I started the publication but then I realized the latex sources are not present in your repo. Can you add them (with the makefile / cls / etc). I'll need to edit it to add the DOI and update volume issue.
(latest template is here: https://github.com/ReScience/template)

In the meantime, can you also save your github repo on software heritage?

**Comment by ValterH** [comment]
> Sorry for long delay, end of year was a bit crazy. 

We understand. Happy New Year!

> I started the publication but then I realized the latex sources are not present in your repo. Can you add them (with the makefile / cls / etc). I'll need to edit it to add the DOI and update volume issue. (latest template is here: https://github.com/ReScience/template)

Done all the source files are in the [rescience directory ](https://github.com/KlemenVovk/noisy-labels/tree/master/rescience). 

> In the meantime, can you also save your github repo on software heritage?

We submitted [a request](https://archive.softwareheritage.org/browse/origin/directory/?origin_url=https://github.com/KlemenVovk/noisy-labels&snapshot=8f9eb576d5247d04ec04cd3c143d3c673f571397). Please let me know if you need some further action on our end.

**Comment by rougier** [comment]
I just uploaded a **sandbox** version at: https://sandbox.zenodo.org/record/423720. Can you check everything is ok? (authors, links, etc). Once you get the swhid I can publish the final version.

**Comment by ValterH** [comment]
Thank you for the quick response.

We found that everything seems to be ok. For the title, however, based on our previous discussion, we agree that a partial reproduction [~Re] would be more appropriate. I see that the pdf I uploaded failed to reflect this change. The updated version now does: https://github.com/KlemenVovk/noisy-labels/blob/master/rescience/rescience.pdf

The SWHID is: `swh:1:dir:22222955c66aeb7e379a16b45d208fcf3d64a11d` at [url](https://archive.softwareheritage.org/browse/origin/directory/?origin_url=https://github.com/KlemenVovk/noisy-labels).

**Comment by rougier** [comment]
Sorry for the delay. Article is online at https://zenodo.org/record/18401497 and shoudl appear  soon on ReScience website.
